DatadogSDK.podspec

```diff
@@ -21,12 +21,11 @@ Pod::Spec.new do |s|
   s.source = { :git => "https://github.com/DataDog/dd-sdk-ios.git", :tag => s.version.to_s }
 
   s.deprecated_in_favor_of =
-    'DatadogCore, DatadogLogs, DatadogRUM, DatadogTrace, and DatadogWebViewTracking'
+    'DatadogCore, DatadogLogs, DatadogRUM, and DatadogTrace'
 
   s.dependency 'DatadogCore', s.version.to_s
   s.dependency 'DatadogLogs', s.version.to_s
   s.dependency 'DatadogRUM', s.version.to_s
   s.dependency 'DatadogTrace', s.version.to_s
-  s.dependency 'DatadogWebViewTracking', s.version.to_s
```

Comment:
question/ What's the explanation of this change 🤔? `DDWVT.podspec` doesn't support `tvOS` hence it failed on `tvOS` compatibility check for `DatadogSDK.podspec`?

Reply:
Exactly! here the checks: 

```
[!] DatadogSDK has been deprecated in favor of DatadogCore, DatadogLogs, DatadogRUM, DatadogTrace, and DatadogWebViewTracking
 -> DatadogSDK (2.0.0-beta1)
    - NOTE  | url: The URL (https://twitter.com/datadoghq) is not reachable.
    - NOTE  | xcodebuild:  note: Using codesigning identity override: -
    - NOTE  | [iOS] xcodebuild:  note: Building targets in dependency order
    - NOTE  | [iOS] xcodebuild:  note: Metadata extraction skipped. No AppIntents.framework dependency found. (in target 'DatadogInternal' from project 'Pods')
    - NOTE  | [iOS] xcodebuild:  note: Metadata extraction skipped. No AppIntents.framework dependency found. (in target 'DatadogWebViewTracking' from project 'Pods')
    - NOTE  | [iOS] xcodebuild:  note: Metadata extraction skipped. No AppIntents.framework dependency found. (in target 'DatadogTrace' from project 'Pods')
    - NOTE  | [iOS] xcodebuild:  note: Metadata extraction skipped. No AppIntents.framework dependency found. (in target 'DatadogLogs' from project 'Pods')
    - NOTE  | [iOS] xcodebuild:  note: Metadata extraction skipped. No AppIntents.framework dependency found. (in target 'DatadogCore' from project 'Pods')
    - NOTE  | [iOS] xcodebuild:  note: Metadata extraction skipped. No AppIntents.framework dependency found. (in target 'DatadogRUM' from project 'Pods')
    - NOTE  | [iOS] xcodebuild:  note: Metadata extraction skipped. No AppIntents.framework dependency found. (in target 'App' from project 'App')
    - ERROR | [tvOS] unknown: Encountered an unknown error (The platform of the target `App` (tvOS 11.0) is not compatible with `DatadogWebViewTracking (2.0.0-beta1)`, which does not support `tvOS`.) during validation.
Analyzed 1 podspec.
```